charts/kyverno-policies/templates/pod-sec-ctxt-capmc.yaml

```diff
@@ -0,0 +1,42 @@
+{{- $name := "pod-sec-ctxt-capmc" }}
+{{- if eq (include "kyverno-policy.podSecurityBaseline" (merge (dict "name" $name) .)) "true" }}
+{{- include "kyverno-policy.supportedKyvernoCheck" (dict "top" . "ver" ">= 1.6.0-0") }}
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  name: {{ $name }}
+  namespace: services
+  annotations:
+    policies.kyverno.io/title: Mutate pod and container security contexts
+    policies.kyverno.io/category: Pod Security Standards (Baseline)
+    {{- if .Values.podSecuritySeverity }}
+    policies.kyverno.io/severity: {{ .Values.podSecuritySeverity }}
+    {{- end }}
+    kyverno.io/kyverno-version: 1.6.0
+    policies.kyverno.io/minversion: 1.6.0
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Highly restrictive mutate policy setting container security context
```

Comment:
Please set specific descriptions and other annotations that you think will help a developer or operator.

Reply:
okay